blog/_posts/2021-07-08-portforwarding-is-awesome.md

```diff
@@ -19,8 +19,23 @@ tools like `cat` for working with files!
 In 2021, we mostly write servers. That's why it's important to have a toolkit for working with 
 network sockets!
 
-In this post, we're going to do a quick tour of the tools we (Tilt-team) work with everyday,
-how we use them, and how we're trying to make it easier to manage them.
+In this post, we're going to do a quick tour of the tools we (the Tilt team) work
+with everyday and how we use them. In particular:
+
+1. `socat`, the assembly language of socket tools.
+
+2. `kubectl port-forward`, an essential building block of any Kubernetes-based dev environment.
+
+3. Tilt's `PortForward` API, a self-healing wrapper around `kubectl port-forward`.
+
+You may use a different set of socket tools! That's OK. 
+
+The goal of this post isn't to try to convince you that these particular tools
+are the best ones. Instead, I've been been thinking through what different tools
+are good for, look at how they interoperate with each other, and come up with a
+framework for figuring out which one is right for what I'm working on.
+
+But if you learn about a new tool in this post, that's fine too!
 
```

Comment:
Yeah, this is great. But we don't have to be so apologetic—how about rephrasing this:

> You may use a different set of socket tools! That's OK. 
> The goal of this post isn't to try to convince you that these particular tools are the best ones. Instead, I've been been thinking through what different tools are good for, look at how they interoperate with each other, and come up with a framework for figuring out which one is right for what I'm working on.
> But if you learn about a new tool in this post, that's fine too!

As:

> I've been thinking through what different tools are good for, looking at how they interoperate with each other, and came up with a framework for figuring out which one is right for what I'm working on. 
> You don't have to use these tools in particular, but I hope you'll learn something new!